nexus/src/main/java/com/example/nexus/model/payload/request/ProductRequest.java

```diff
@@ -26,6 +26,10 @@ public record ProductRequest(
         @PositiveOrZero(message = MessageConstants.INVALID_AVAILABILITY)
         int availability,
 
+        @Min(value = 0, message = MessageConstants.INVALID_DISCOUNT)
+        @Max(value = 100, message = MessageConstants.INVALID_DISCOUNT)
+        Integer discount,
```

Comment:
Add @RequestParam(required = false) because when create product we dont need to set discount